components/internal_server/src/quality_time_server.py

```diff
@@ -13,13 +13,14 @@
 
 from shared.initialization.database import init_database  # skipcq: FLK-E402
 
-from initialization import init_bottle  # skipcq: FLK-E402
+from initialization import init_bottle, merge_unmerged_measurements  # skipcq: FLK-E402
 
 
 def serve() -> None:  # pragma: no cover-behave
     """Connect to the database and start the application server."""
     logging.getLogger().setLevel(logging.INFO)
     database = init_database()
+    merge_unmerged_measurements(database)
```

Comment:
Why not register in the database that this code has run?
This call could then be made conditional to absence of that registration.

Reply:
Good question. The advantage is that the code does not run multiple times. The disadvantage is that we need to set a flag which pollutes the database and/or would need to be removed at a later time. 

The migration only runs on startup and when there is nothing to do, should be pretty quick and not write anything to the database. If it takes more than a few seconds on our biggest database, we'll consider adding the flag.